### ISSUE TITLE
How to get the current value of the fuse change from the FusesSet event log?

### ISSUE THREAD

**Issue by brucexc** [ensdomains/ens-contracts#249]
After the ENS FusesSet event log has been cleaned, it is no longer possible to retrieve the current value of the fuse change from the log.

Is there any solution to this problem?

Related event log link: https://etherscan.io/tx/0xf0d11296e8954146a7e7e21ce2f17f120b72af134194412931bb4402487dcab4#eventlog

**Comment by Arachnid** [comment]
Do you mean the value before the change? You're right, the log only contains the value after the change.

**Comment by brucexc** [comment]
> Do you mean the value before the change? You're right, the log only contains the value after the change.

Yes. Including the value before the change and/or the changed value in the event log would be beneficial.

**Comment by Lalith514** [comment]
yes

Historical Data: If you have access to a historical data service or blockchain explorer that records and archives all events, you might be able to retrieve the past logs from there. These services often store historical data, allowing you to access events that occurred before the log was cleaned.
It's important to note that the availability and feasibility of these solutions depend on the specific circumstances and infrastructure of your system.

**Comment by Mrt3383x38** [comment]
Benim  giden saglıgımı geri verebilir mi ölmek üzere olan anneme kalp 💓 verebilirim dört yıldır bı kuru canımız kaldı Allah yaşattığı nızı yaşayın?
